src/modules/ethereum/chains/ChainConfigStore.ts

```diff
@@ -61,10 +98,41 @@ export class ChainConfigStore extends PersistentStore<ChainConfig> {
     const chainStr = chain.toString();
     this.setState((state) =>
       produce(state, (draft) => {
-        remove(draft.ethereumChains, (x) => x.value.chain === chainStr);
+        if (!draft.ethereumChainConfigs) {
+          draft.ethereumChainConfigs = [];
+        }
```

Comment:
I don't think this check is needed anymore, it's guaranteed to exist

Reply:
Yes! I fixed this whole thing